src/test/java/org/myrobotlab/service/ServoTest.java

```diff
@@ -195,40 +195,44 @@ public void releaseService() {
 
   @Test
   public void testAutoDisable() throws Exception {
-    // Start the test servo.
-    servo01.detach();
-    servo01.setPin(pin);
-    servo01.setPosition(0.0);
+    // Start the test servo
+    arduino01 = (Arduino) Runtime.start(arduino01.getName(), "Arduino");
+    arduino01.connect(port01);
+    servo.detach();
+    sleep(300);
```

Comment:
this isn't good to have this sleep here.. servo.detach() should probably be blocking/synchronous....  otherwise we'll end up with race conditions elsewhere.

Reply:
perhaps this detach is only an issue in virtual land..